### PR TITLE
Update auditing chapter

### PR DESCRIPTION
**Pull request by ares**
@tbrisker we discussed this content few months ago, mind to take a look?

### CODE REVIEW

File: _includes/manuals/nightly/4.1.4_auditing.md [hunk 1]
```diff
@@ -48,4 +48,12 @@ Example:
 
 1. Anonymizes all audits older then 7 days
 
-		<pre>foreman-rake audits:anonymize days=7</pre>
\ No newline at end of file
+		<pre>foreman-rake audits:anonymize days=7</pre>
+
+#### Organizations and Locations
+
+Audits inherit organizations and locations of resources for which they have been created. Imagine you have a subnet assigned to organization A. Whenever you modify this subnet, the audit will be visible only in organization A. When you later add this subnet to organization B, new audits will appear in both organizations A and B. All audits created previously remain untouched. Resources that can't be assigned to any organization or location will be allways visible in all organizations and locations, meaning the change has affected all organizations and locations.
```
Comment (tbrisker) [inline _includes/manuals/nightly/4.1.4_auditing.md]:
```suggestion
Audits inherit organizations and locations of resources for which they have been created. Imagine you have a subnet assigned to organization A. Whenever you modify this subnet, the audit will be visible only in organization A. When you later add this subnet to organization B, new audits will appear in both organizations A and B. All audits created previously remain untouched. Resources that can't be assigned to any organization or location will be always visible in all organizations and locations, meaning the change has affected all organizations and locations.
```

**Review by tbrisker — CHANGES_REQUESTED** [review]
just a tiny type (allways->always). Can you please update the 1.20 manual with the same?

**Comment by ares** [comment]
thanks, updated

**Review by tbrisker — APPROVED** [review]
Thanks @ares !

**Pull request merged**